### PR TITLE
Try to get of Pypy on pull-requests

### PR DESCRIPTION
**Pull request by leplatrem**
Pypy builds take up to 25min. I like the idea of supporting PyPy, but maybe not up to this point!
So let's try keep Pypy tests for master only.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,3 +39,7 @@ matrix:
       install: pip install flake8
       before_script: echo 'flake8'
       script: flake8 kinto
+  exclude:
+    # PyPy test suite takes around 20min.
+    - TOX_ENV: pypy
+      TRAVIS_PULL_REQUEST: true
```
Comment (Natim) [inline .travis.yml]: Missing blank line at the end of the file :0

File: .travis.yml [hunk 2]
```diff
@@ -37,3 +38,5 @@ matrix:
       install: pip install flake8
       before_script: echo 'flake8'
       script: flake8 kinto
+    - env: TOX_ENV=pypy
+      script: if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then tox -e $TOX_ENV; fi;
```
Comment (Natim) [inline .travis.yml]: It doesn't looks to be enough.

**Comment by leplatrem** [comment]
capture d ecran de 2016-07-29 17-22-43

**Comment by leplatrem** [comment]
There are two jobs: one with the merge on the targetted branch, and one for the branch.
The current version of the patch removes pypi on one of them only.

**Pull request merged**